examples/EchoServer.cpp

```diff
@@ -12,11 +12,17 @@ int main() {
 
     /* Keep in mind that uWS::SSLApp({options}) is the same as uWS::App() when compiled without SSL support.
      * You may swap to using uWS:App() if you don't need SSL */
-    uWS::SSLApp({
+    uWS::App({
```

Comment:
this was so I could run the EchoServer without HTTPS, but there was no need to really commit this

Reply:
yep, examples won't change